docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -13,6 +13,8 @@ endif::notBuildingForSite[]
 
 == Introduction
 
+Distributed Caching can be enabled on a cache by adding Clustered storage as a tier to the cache.
```

Comment:
I think this addition makes sense but then the whole section needs to be reworked. Whatever the order, there are two separate things:
* Have a started server
* Have the proper client configuration
  * Cache manager
  * Cache with clustered tier
And maybe before that there can be a reference to the benefits of clustering.